node/node/Cargo.toml

```diff
@@ -69,6 +69,7 @@ thiserror = { workspace = true }
 time-source = { workspace = true }
 derive-new = { workspace = true }
 partner-chains-node-commands = { workspace = true }
+partner-chains-smart-contracts-commands = { workspace = true }
```

Comment:
To be honest, I've expected that `partner-chains-node-commands` will bring it and users will need one our crate for commands, not two

Reply:
it does. I removed this dependency